odd-platform-api/src/main/resources/application.yml

```diff
@@ -1,8 +1,8 @@
 spring:
   datasource:
-    url: jdbc:postgresql://127.0.0.1:5432/odd
-    username: odd
-    password: odd_password
+    url: jdbc:postgresql://127.0.0.1:5432/odd-platform
```

Comment:
Are these intended changes? Mine works fine on `main` branch.

Reply:
They are, however I didn't change dev's folder settings. Fixed. It's done for consistency